spec/spec.md

```diff
@@ -284,6 +284,10 @@ DIDComms   | `$.presentations~attach.data.json`
 VP         | top-level
 CHAPI      | `$.data`
 
+### Input Evaluation
+
+A conformant consumer of a [[ref:Credential Manifest]] containing a `presentation_definition` property ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) must be constructed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```

Comment:
```suggestion
A conformant consumer of a [[ref:Credential Manifest]] containing a `presentation_definition` property ****MUST**** include a valid [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) in the  `presentation_submission` property of its corresponding [[ref:Credential Application]]. The [Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) must be processed as specified in [Presentation Exchange: Input Evaluation](https://identity.foundation/presentation-exchange/#input-evaluation).
```

Reply:
Note to self: is conformant consumer the right term?

Reply:
`processor`?

Reply:
`applicant`?

Reply:
one other comment - there are two types of processing
1. the applicant needs to process the manifest's submission and generate a compliant submission as a part of their application
2. the creator of the manifest (issuer?) needs to then process the credential application's presentation submission according to the pe spec